validator/src/main/java/com/google/daq/mqtt/registrar/LocalDevice.java

```diff
@@ -112,12 +113,17 @@
   );
 
   private static final Set<String> OPTIONAL_FILES = ImmutableSet.of(
-      RSA2_PUBLIC_PEM, RSA3_PUBLIC_PEM, ES2_PUBLIC_PEM, ES3_PUBLIC_PEM,
+      RSA_PUBLIC_PEM, RSA2_PUBLIC_PEM, RSA3_PUBLIC_PEM, 
+      ES_PUBLIC_PEM, ES2_PUBLIC_PEM, ES3_PUBLIC_PEM,
       GENERATED_CONFIG_JSON, DEVICE_ERRORS_JSON, NORMALIZED_JSON, SAMPLES_DIR);
   private static final Set<String> ALL_KEY_FILES = ImmutableSet.of(
       RSA_CERT_PEM, RSA_PUBLIC_PEM, RSA2_PUBLIC_PEM, RSA3_PUBLIC_PEM,
       ES_CERT_PEM, ES_PUBLIC_PEM, ES2_PUBLIC_PEM, ES3_PUBLIC_PEM
   );
+  private static final Set<String> ALL_CERT_FILES = ImmutableSet.of(
+      RSA_CERT_PEM, 
```

Comment:
Do these values need to still be in ALL_KEY_FILES, or can/should they be removed from there?

Reply:
They need to be there.